public/content/community/grants/index.md

```diff
@@ -18,7 +18,6 @@ These programs support the broad Ethereum ecosystem by offering grants to a wide
 - [MetaCartel](https://www.metacartel.org/grants/) - _Dapp development, DAO creation_
 - [Moloch DAO](https://www.molochdao.com/) - _Privacy, layer 2 scaling, client security, and more_
 - [DAO Grants](https://docs.google.com/spreadsheets/d/1XHc-p_MHNRdjacc8uOEjtPoWL86olP4GyxAJOFO0zxY/edit#gid=0) - _Google spreadsheet of organizations offering grants_
-- [Crunchbase for Web3 Grants](https://www.cryptoneur.xyz/web3-grants) - _Filter and search for grants by category, use case, amount, and more. Contribute to help others find the right grant._
```

Comment:
Yeah looks like this isn't available now:

<img width="1265" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/6a5cc4af-e9a4-49d8-b168-c5adac8af82b">